.github/workflows/publish-native-assets-to-github-releases.yml

```diff
@@ -5,6 +5,7 @@ name: Add native binaries to release
 on:
   release:
     types: [created]
+  workflow_dispatch:
```

Comment:
What does this do?

Reply:
It allows manual triggering of this workflow, not only when release is created.